### DISCUSSION TITLE
Update question

### DISCUSSION THREAD

**Discussion by roswitina** [ej52/proxmox-scripts#27] [Q&A]
Is it possible to update the version with the script?

**Comment by ej52** [comment]
Yes, if you run `wget --no-cache -qO - https://raw.githubusercontent.com/ej52/proxmox/main/lxc/nginx-proxy-manager/setup.sh | sh` in the container, it will update to the latest version while keeping your settings.

**Comment by DinkyToys** [comment]
When you execute this command for NPM 2.9.15 version, you will receive error messages with Yarn and that Linux Distro does not support. After this you will receive a login screen, but logging in doesn't work. Control via API / Tokens you get a 502 error. Please a solution for updating NPM.
